python/tests/algorithms/ml/test_arrival.py

```diff
@@ -0,0 +1,82 @@
+import numpy as np
+import sys
+
+import seisbench.models as sbm
+from mspasspy.algorithms.ml.arrival import annotate_arrival_time
+from mspasspy.ccore.algorithms.basic import TimeWindow
+from mspasspy.util.converter import Trace2TimeSeries
+from obspy import Stream, UTCDateTime
+from obspy.clients.fdsn import Client
+
+sys.path.append("python/tests")
+
+from helper import get_live_timeseries
+
+pn_model = sbm.PhaseNet.from_pretrained("stead")
+
```

Comment:
Maybe we should add a test with real data. We do have some real waveform data in the [data dir](https://github.com/mspass-team/mspass/tree/master/python/tests/data), and you might want to reuse one of them for the test. I think this will be helpful with the relative time vs absolute time confusion I had above. Also, please add some necessary comments to the test code too - I know we were not doing great commenting the test code in the existing code base, but we should improve it from now on.

Reply:
sure, I can use real data to test and provide more comments for the test cases.

are you aware of any existing code that imports those data into mspass formats? that would be really helpful to me

Reply:
https://docs.obspy.org/packages/autogen/obspy.io.mseed.core._read_mseed.html